care.js

```diff
@@ -1,180 +1,167 @@
 #!/usr/bin/env node
-var config = require(__dirname + '/config.js');
-var twitterbot = require(__dirname + '/twitterbot.js');
-
-var spawn = require( 'child_process' ).spawn;
-var blessed = require('blessed');
-var contrib = require('blessed-contrib');
-var chalk = require('chalk');
-var parrotSay = require('parrotsay-api');
-var weather = require('weather-js');
-
-var screen = blessed.screen(
-    {fullUnicode: true, // emoji or bust
-     smartCSR: true,
-     autoPadding: true,
-     title: '✨💖 tiny care terminal 💖✨'
-    });
+const config = require(__dirname + '/config.js')
```

Comment:
Sorry, but I'm really keen on semicolons. Can you add them back? (here and everywhere)